### DISCUSSION TITLE
Question on ELF support

### DISCUSSION THREAD

**Discussion by jchap2k** [mariusmm/RISC-V-TLM#14] [General]
Hi Mariusmm,

Is there any update on the ability to load an ELF file into memory?  I am looking at getting your RISCV model imported into Synopsys VDK, and it will be possible to debug in the integrated debugger if I am able to load ELFs instead of hex.  (I can always follow your instructions on how to get debug working with hex, but that process isn't as clean and easy to propagate to others)

John

**Comment by mariusmm** [comment]
Hi @jchap2k !

Thank you for your interest on this simulator.
It was not planned to add elf support, but if there is some interest on this feature I can try to implement it.
Can you explain your work on RISC-V-TLM and Synopsys VDK?

Thanks!

Màrius
